chapters/synchronous.tex

```diff
@@ -737,18 +753,11 @@ \subsection{Sub-clock conversion operators}\doublelabel{sub-clock-conversion-ope
 \textbf{shiftSample}(u,\\
 shiftCounter, resolution)
 \end{tabular}
-& {[}\emph{The first activation of the clock of y =
-\textbf{shiftSample}(..) is shifted in time
-shiftCounter/resolution*interval(u) later than the first activation of
-the clock of u.}{]}.
-
-Conceptually, the operator constructs a clock ``cBase''
+& The operator \lstinline!shiftSample(u,k)! skips the first k ticks of u, and is then returning every tick.
+With resolution it is defined as:
```

Comment:
I think this is not sufficient. The fractional shifts of wrt resolution are really non trivial. We should find a prosa expression for that too. Maybe something along the lines of:

>The operator \lstinline! c=shiftSample(u,k,resolution)! shifts the first activation of the clock c by `k * interval(u) / resolution` relative to the first activation of `u`.  The clock `c` is then activating with the same frequency as `u`.

Reply:
Maybe even better:

> The operator \lstinline! c=shiftSample(u,k,resolution)! splits the interval between activations of the clock of u into `resolution` equidistant intervals `i`. The clock `c` then activates `k` intervals `i` after each activation of `u`.

Reply:
Looks good, but I don't like having two normative explanations for the same thing.
Should we have this as non-normative or the equation-form as non-normative?